Drivers/Inc/CANbus.h

```diff
@@ -29,7 +29,7 @@
  */
 typedef enum { 
 	BPS_TRIP						= 0x002,
-	BPS_CONTACTOR					= 0x101,
+	BPS_CONTACTOR					= 0x102,
```

Comment:
why did this can id change / is it documented in the sheet properly

Reply:
It is updated in the sheet. @IshDeshpa Why did this change?

Reply:
software typo. It is correct